setup.py

```diff
@@ -6,7 +6,7 @@
 
 setup(
     name="descript-audiotools",
-    version="0.7.3",
+    version="0.7.4",
```

Comment:
update it here as well:
https://github.com/descriptinc/audiotools/blob/master/audiotools/__init__.py#L1